crates/quickjs-wasm-rs/src/js_value/qjs_convert.rs

```diff
@@ -22,10 +22,10 @@ use crate::js_binding::{context::JSContextRef, value::JSValueRef};
 ///
 /// ```
 /// // Assuming `qjs_val` is a `JSValueRef` pointing to a QuickJS string
-/// let js_val = from_qjs_value(&qjs_val).unwrap();
+/// let js_val = from_qjs_value(qjs_val).unwrap();
 /// assert_eq!(js_val, "hello".into());
 /// ```
-pub fn from_qjs_value(val: &JSValueRef) -> Result<JSValue> {
+pub fn from_qjs_value(val: JSValueRef) -> Result<JSValue> {
```

Comment:
Can you update the comment on line 11 to indicate it converts the ref and not a reference to it?